src/library_syscall.js

```diff
@@ -458,7 +458,15 @@ var SyscallsLibrary = {
       FS.munmap(stream);
       SYSCALLS.mappings[addr] = null;
       if (info.allocated) {
+#if USE_LSAN
+        if (info.anonymous) {
```

Comment:
this is not very obvious. how about a comment explaining why we need to do this?